src/cowboy_protocol.erl

```diff
@@ -361,7 +361,13 @@ parse_hd_value(<<>>, S, M, P, Q, V, H, N, SoFar) ->
 request(B, State=#state{transport=Transport}, M, P, Q, Version, Headers) ->
 	case lists:keyfind(<<"host">>, 1, Headers) of
 		false when Version =:= 'HTTP/1.1' ->
-			error_terminate(400, State);
+			%% Advenias:
+			%% Why: AWS Application Load Balancer's health check seems to respond with empty `Host:`
```

Comment:
Not 'empty' Host, but rather 'missing' Host, big differences between the two as one follows the spec and the other does not.